### DISCUSSION TITLE
Consider uploaded images from phone as uploaded

### DISCUSSION THREAD

**Discussion by datmospro** [immich-app/immich#8749] [General]
If i delete from WEB a photo uploaded frome phone, it gets deleted on web and on the APP, but not on the phone gallery, so after a wile, the same photo will be uploaded again.

Is there any way to mark a photo as "uploaded" and never try to upload it again automatically? Only in some manual way yes.

Thanks!

**Comment by RafaelSchridi** [comment]
Was also looking for something like this, I've uploaded all my files using CLI and now I want to mark all my pictures on my phone as uploaded.

I previously used PhotoSync for this which marked all photos send as uploaded unless they changed.
